examples/pypi/run.sh

```diff
@@ -0,0 +1,22 @@
+set -x
+set -e
+
+# Start artipie.
+docker run --name artipie -d -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Build and upload python project to Artipie
+cd sample-project
+python3 -m pip install --user --upgrade setuptools wheel
+python3 setup.py sdist bdist_wheel
+python3 -m pip install --user --upgrade twine
+python3 -m twine upload --repository-url http://localhost:8080/my-pypi -u username -p 123 dist/*
```

Comment:
Is possible to get rid of passing credentials here? Since we permit anonymous access in the example, I guess we need to demonstrate it.

And even if you decide to keep the credentials, please, make it consistent with the README.md. In the README.md you use a different password: `-p password`

Reply:
Unfortunately, credentials are required by [twine](https://twine.readthedocs.io/en/latest/#commands): we can either pass them like it's done here or set via environment variables, I think first option is better in our case. 

Also I've corrected password value to be consisted with readme.